uiowa/uids
# fixing table width

This PR should reset all tables to a default of 100% width, with `table--width-default` opting out of the 100% width.

# To test
1. Pull down this branch
2. run `npm run watch` to spin up the server
3. check that if a table is given `table--static` it will be 100% width until it is given `table--width-default`
